tapscript/send.go

```diff
@@ -502,17 +502,18 @@ func PrepareOutputAssets(ctx context.Context, vPkt *tappsbt.VPacket) error {
 		// TODO(ffranr): Add support for interactive full value multiple
 		// input spend.
 		input := inputs[0]
+		vOut := outputs[recipientIndex]
 
 		// We'll now create a new copy of the old asset, swapping out
 		// the script key. We blank out the tweaked key information as
 		// this is now an external asset.
-		outputs[recipientIndex].Asset = input.Asset().Copy()
-		outputs[recipientIndex].Asset.ScriptKey = outputs[0].ScriptKey
```

Comment:
As for burns we always have the burn output be second?

Reply:
For sake of time, can you add some more detail to the commit body re the scenario?

Reply:
Makes sense, added more context to the commit message.